src/erlfmt.erl

```diff
@@ -80,37 +84,73 @@ format_file(FileName, {Pragma, Out}) ->
 
 -spec format_string(string(), integer()) -> string().
 format_string(String, PageWidth) ->
+    format_string(String, PageWidth, ignore).
```

Comment:
Do we need to keep the `format_string/2` function around? I'd be fine only having the `/3` variant. For formatting file we don't have a version without options.

Reply:
removed :)